plugins/mcreator-localization/lang/texts.properties

```diff
@@ -783,13 +783,15 @@ trigger.player_completes_advancement=Player completes advancement
 trigger.player_critical_hit=Player deals critical hit
 trigger.player_fishes_item=Player fishes item
 trigger.player_in_bed=Player goes to bed
+trigger.player_left_click=Player left clicks
 trigger.player_left_click_block=Player left clicks block
 trigger.player_log_in=Player joins the world
 trigger.player_log_out=Player leaves the world
 trigger.player_respawn=Player respawns
 trigger.player_right_click_block=Player right clicks block
 trigger.player_right_click_entity=Player right clicks on entity
 trigger.player_right_click_item=Player right clicks with item
+trigger.player_right_click_empty_hand=Player right clicks with empty hand
```

Comment:
```suggestion
trigger.player_right_click_empty_hand=Player right clicks in air
```

Reply:
Just fixed this to actually only trigger with an empty hand, once. So no need for it.